com.github.youwin.VPupPr.yml

```diff
@@ -0,0 +1,49 @@
+id: com.github.youwin.VPupPr
```

Comment:
this app-id doesn't match the project below, or the URLs.

Reply:
the id can't have the you-win part in the name, unless it's on the very last part. Maybe I could replace it with an underscore? or do you mean the capitalization of the last part of the name?

Reply:
According to [this answer of smcv](https://github.com/flatpak/flatpak/issues/4974#issuecomment-1169862657) you should use a underscore

Reply:
I've renamed every instance I can find of youwin to you_win